### DISCUSSION TITLE
Upgrading an Piccap setup which didn't really work some questions which parts

### DISCUSSION THREAD

**Discussion by FTBSchnitzel** [awawa-dev/HyperHDR#959] [Q&A] [answered]
I have right now BTF-LIGHTING WS2812B a pi 3 a 5V Netzteil 10A 50W psu which are connected to the pi using microusb and the led using a barrel jack terminal. I had problems with piccap that it was disconnecting so i want to switch to using a led driver with 3.3 to 5v The tv is a lg 4k 50hz display. The problem idk which is a good switcher driver and capture card. I need a minimum of 2 inputs and for shipping would be the eu the best as I live in germany. Aliexpress would work too. Also which segment system i should use and how i should connect them with a rp2040 with 3.3 to 5v shifter this is about the setup i have right now without the switcher and capture card https://www.youtube.com/watch?v=J26oYlKyq7Q

**Comment by awawa-dev** [comment]
From what I know most users in similar setups install HyperHDR directly on the TV instead of the RPi3. They add to this usually WLED or for example: Adafruit Scorpio rp2040 as an LED wired controller connected to the TV USB port. The minus from the LED power supply in this case is connected to both the LED and the Scorpio and the plus only to the LED strip.

**Comment by awawa-dev** [comment, marked as answer]
https://github.com/satgit62/How-to-Install-and-set-up-Ambilight-on-LG-webOS
